packages/search-ui/src/SearchDriver.js

```diff
@@ -235,13 +248,29 @@ export default class SearchDriver {
         if (this.requestSequencer.isOldRequest(requestId)) return;
         this.requestSequencer.completed(requestId);
 
+        // Results/Paging start & end
+        const { totalResults } = resultState;
+        const start =
+          totalResults === 0 ? 0 : (current - 1) * resultsPerPage + 1;
+        const end =
+          totalResults <= start + resultsPerPage
+            ? totalResults
+            : start + resultsPerPage - 1;
```

Comment:
Following up here on our decision to move `start`/`end` logic out of `PagingInfoContainer` and into `SearchDriver` - are you guys good with this? Or is there a more elegant way/place to put this that I'm missing? 🤔 

Here's the full commit for context: https://github.com/elastic/search-ui/pull/355/commits/4c58a7381f21174a205f61e1ae51a31744e65a57

Reply:
@constancecchen This looks correct to me. Since these have moved to the global state, I think these should be renamed to give some more context. Maybe `pagingStart` and `pagingEnd`?